aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -277,4 +300,23 @@ module aptos_framework::code {
         bundle: vector<vector<u8>>,
         policy: u8
     );
+
+    /// A helper type for request_publish_with_allowed_deps
+    struct AllowedDep has copy, drop, store {
```

Comment:
why have store or copy, I don't think I saw either being used?

Reply:
Why not?

Reply:
But a good point -- fixed.